File: CS/Exercises/Exercise1/File.cs
using System.IO;

namespace Exercises.Exercise1;

static class File
{
  public static void Write(string path, string contents)
  {
    Directory.CreateDirectory(Path.GetDirectoryName(path)!);
    System.IO.File.WriteAllText(path, contents);
    Thread.Sleep(4000);
  }
}

File: VB/Exercises/Exercise1/File.vb
Imports System.Threading

Namespace Exercises.Exercise1
  Module File
    Public Sub Write(path As String, contents As String)
      IO.Directory.CreateDirectory(IO.Path.GetDirectoryName(path))
      IO.File.WriteAllText(path, contents)
      Thread.Sleep(4000)
    End Sub
  End Module
End Namespace
